.docker/Crawler/Dockerfile

```diff
@@ -10,6 +10,4 @@ ADD ./run-crawler.sh /Arriba/Tools/app
 WORKDIR /Arriba/Tools/app
 RUN chmod +x ./run-crawler.sh
 
-RUN ls -l
-RUN find /Arriba
-CMD ["./run-crawler.sh", "configName=default", "mode=-i"] 
\ No newline at end of file
+CMD ["./run-crawler.sh"] 
```

Comment:
I would prefer to use:

```
ENTRYPOINT ["./run-crawler.sh"]
CMD ["configName=default", "mode=-i"]
```

This way we can pass a different configuration, but yet having a default value.